Fruit_Jam/Larsio_Paint_Music/sound_manager.py

```diff
@@ -98,9 +106,24 @@ def __init__(self, audio_output="pwm", seconds_per_eighth=0.25):
 
                 # Initialize TLV320
                 self.tlv = adafruit_tlv320.TLV320DAC3100(i2c)
+                self.tlv.reset()
+
+                # set sample rate & bit depth
                 self.tlv.configure_clocks(sample_rate=11025, bit_depth=16)
-                self.tlv.headphone_output = True
-                self.tlv.headphone_volume = -15  # dB
+
+                if "tlv320" in launcher_config:
+                    if launcher_config["tlv320"].get("output") == "speaker":
+                        # use speaker
+                        self.tlv.speaker_output = True
+                        self.tlv.dac_volume = launcher_config["tlv320"].get("volume",5)  # dB
+                    else:
+                        # use headphones
+                        self.tlv.headphone_output = True
+                        self.tlv.dac_volume = launcher_config["tlv320"].get("volume",0)  # dB
+                else:
+                    # default to headphones
+                    self.tlv.headphone_output = True
+                    self.tlv.dac_volume = 0  # dB
```

Comment:
does this end up playing louder than the previous sound level with `-15` on headphone_volume?